build.gradle

```diff
@@ -65,9 +68,35 @@ test {
     }
 }
 
-tasks.withType(JavaCompile) {
+def webViewVersion = '0.1.14'
+def webviewFileName = 'jfrog-ide-webview-' + webViewVersion + '.tgz'
+tasks.register('downloadWebview', Download) {
+    src 'https://releases.jfrog.io/artifactory/ide-webview-npm/jfrog-ide-webview/-/' + webviewFileName
+    dest buildDir
+    onlyIfModified true
+    finalizedBy('verifyWebview')
+}
+
+tasks.register('verifyWebview', Verify) {
+    src new File(buildDir, webviewFileName)
+    algorithm 'SHA-512'
+    checksum 'b2015294714222de1efa2490e9d1b8d36fd82aa79a4d04fd0821c07da8799eb85d45f1dedede2366da71b2d91f6277e31ce7befc9c4cbe2229b437d731fc7689'
```

Comment:
If the checksum does not match, it returns an error. If the webview version has been updated or it is the first time running this task, an error occurs.

Reply:
We (me and Or) agreed not to change this logic at this time, since we didn't found more secured solution